cs/src/Management/TunnelManagementClient.cs

```diff
@@ -458,8 +507,9 @@ private string UserLimitsPath
             where TRequest : class
         {
             var uri = BuildUri(clusterId, path, query, options);
+            Tunnel? tunnel = null;
             var authHeader = await GetAuthenticationHeaderAsync(
-                tunnel: null, accessTokenScopes: null, options);
+                tunnel: tunnel, accessTokenScopes: null, options);
```

Comment:
Why changing from `null` to `tunnel`? It'll change how the auth header is calculated.

Reply:
Actually, I see it's `null` anyway.

Reply:
It makes sure it can differentiate between the functions